ta/pkcs11/src/persistent_token.c

```diff
@@ -128,6 +128,160 @@ enum pkcs11_rc verify_pin(enum pkcs11_user_type user, const uint8_t *pin,
 	return rc;
 }
 
+enum pkcs11_rc setup_so_identity_auth_from_client(struct ck_token *token)
+{
+	TEE_Identity identity = { 0 };
```

Comment:
Ok. I was looking some other reviews and they wanted to have 0 in there so that was the last step I modified in this PR before sending it out ;)

Can remove.

Reply:
Removed.